include/hstcal.h

```diff
@@ -8,8 +8,8 @@
 //
 // Note: IRAF_SZ_LINE = 1023 relates to the maximum filename size accepted by cfitsio.
 #define CHAR_FNAME_LENGTH 255
-#define CHAR_LINE_LENGTH 255
-#define MSG_BUFF_LENGTH CHAR_LINE_LENGTH + 1
+#define CHAR_LINE_LENGTH 1023
+#define MSG_BUFF_LENGTH (CHAR_LINE_LENGTH * 2 + 1)
```

Comment:
Do we need to add comment to explain why these numbers are chosen?

Reply:
Originally I increased the size because if you trigger several errors at once it would overflow the buffer. I chose the values based on the commented values above, and the file commit history. It used to be 1023 anyway.

The buffer length is padded to 2kb to help avoid overflowing the global buffer when  `strcat` and `sprintf` (there are many) inject long strings. This isn't foolproof of course. One can still cause an overflow when conditions are right, but now it takes significantly more effort to make it happen. 

Using the `n` variants of the string op functions whenever possible will further reduce this risk.

Reply:
1023 * 2 + 1 = not power of 2

👹

Reply:
Oh wait but if you + 1 after it, it is. OK OK we good now.